packages/fe-fpm-writer/test/integration/__snapshots__/basic-lrop-app.test.ts.snap

```diff
@@ -450,6 +496,21 @@ Object {
                       \\"enabled\\": true,
                       \\"text\\": \\"Navigate to CustomPage (appended action)\\"
                     }
+                  },
+                  \\"facets\\": {
+                    \\"MyCustomHeaderSection\\": {
+                      \\"template\\": \\"v4travel.ext.myCustomHeaderSection.MyCustomHeaderSection\\",
+                      \\"templateEdit\\": \\".MyCustomHeaderSectionEdit\\",
```

Comment:
`templateEdit` value looks wrong. I think it should be `v4travel.ext.myCustomHeaderSectionEdit.MyCustomHeaderSectionEdit`

Reply:
fix - 4c6fcdda05fc483345b76f297408a89e82084633